### PR TITLE
COVID-19 Metrics API

### PR DESCRIPTION
**Pull request by ahmetsenturk**
Covid-19 metrics functionality works just fine, it gets the covid-19 metrics for Story's location from an API and returns it in JSON format.

Unit tests are also implemented to check the program.

### CODE REVIEW

**Review by niyaziulke — CHANGES_REQUESTED** [review]
Many binary files are pushed: pycache, database, etc.

File: practice-app/api/urls.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
-from django.urls import path, include
+from django.urls import path
 from . import views
 
-
-
 urlpatterns = [
+    path('posts/covid/<int:post_id>', views.get_covid_numbers, name='covid_numbers'),
```
Comment (mertlkn) [inline practice-app/api/urls.py:5]: I think parameter here should be story_id. I could not run the tests with post_id.

Comment (ahmetsenturk) [reply practice-app/api/urls.py:5]: Thank you for spotting the mistake here, fixing it immediately!

**Comment by uskudarli** [comment]
Already mentioned, alas make sure to use .gitignore to avoid committed system files and config files that have sensitive info,

**Review by zcanfes — COMMENTED** [review]
I think it looks good. You have implemented the necessary functionalities and tested your code accordingly. Comments are well written and exceptions are handled properly.

**Review by melihaydogd — APPROVED** [review]
If you remove 0001_initial.py and binary files. Your api functionality is ok to go.

**Review by melihozcan1 — APPROVED** [review]
Your API seems to be functioning according to what it's supposed to do.

**Review by mertlkn — APPROVED** [review]

**Review by niyaziulke — APPROVED** [review]

**Review by omarr09 — APPROVED** [review]
Getting COVID-19 data of the story location is a very good idea. And the code looks good too

**Comment by kkadirkkalkan** [comment]
Looks fine. Code is understandeble.

**Review by mecolak — APPROVED** [review]
Looks good!

**Review by kkadirkkalkan — APPROVED** [review]

**Pull request merged**